### ISSUE TITLE
Delete Signs/Markings Objects and Views

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-knack-data-tracker#163]
Only after `Project: Signs Migration` is complete.

**Comment by amenity** [comment]
@dianamartin @johnclary - do we still need this?

**Comment by dianamartin** [comment]
@amenity I just checked the Data Tracker and I think we've already done this.
